File: src/main/java/cc/onelooker/kaleido/third/tmm/Episode.java
package cc.onelooker.kaleido.third.tmm;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

/**
 * @Author xiadawei
 * @Date 2024-02-02 13:53:00
 * @Description TODO
 */
@Data
public class Episode {

    @JsonProperty("tmdb_id")
    private String tmdbId;
    @JsonProperty("episode_number")
    private Integer episodeNumber;
    @JsonProperty("season_number")
    private Integer seasonNumber;
    private String title;
    private String plot;
    private String premiered;
    private String year;
    private Float average;
    private Integer votes;
    private Integer runtime;
    private String thumb;

}

File: web-scraper/crawler/entity/episode.py
from dataclasses import dataclass, field


@dataclass
class Episode:
    tmdb_id: str = None
    premiered: str = None
    year: str = None
    episode_number: str = None
    season_number: str = None
    title: str = None
    plot: str = None
    average: str = None
    votes: str = None
    runtime: str = None
    thumb: str = None
